test/cppsuite/configs/cursor_bound_01_default.txt

```diff
@@ -0,0 +1,58 @@
+# Configuration for cursor_bound_01 default test.
+# During the test duration:
+#   - M threads will keep inserting new random keys.
+#   - N threads will execute search_near calls with random bounds set. Each search_near
+# call with bounds set is verified against the default search_near.
+#   - O threads will continuously remove random keys.
+#   - P threads will continuously update random keys.
+#   - Q threads will utilize the custom operation and will execute next() and prev() calls with
+# random bounds.
+duration_seconds=60,
```

Comment:
Make this 20 seconds, the more cppsuite tests we add the more PR testing will slow down.